actions/ingestion/webhook/index.js

```diff
@@ -76,22 +68,27 @@ async function main (params) {
 
     logger.info('Process event data')
     logger.debug(
-          `Process event ${event.type} for entity ${event.entity}`)
+          `Process event ${params.data.event}`)
 
     const cloudEvent = new CloudEvent({
-      source: 'urn:uuid:' + event.providerId,
-      type: event.type,
+      source: 'urn:uuid:' + provider.id,
+      type: params.data.event,
       datacontenttype: 'application/json',
-      data: event.data,
+      data: params.data.value,
       id: uuid.v4()
     })
 
-    logger.debug(`Publish event ${event.type} to provider ${event.providerName}`)
-    event.success = await eventsClient.publishEvent(cloudEvent)
+    logger.debug(`Publish event ${params.data.event} to provider ${provider.label}`)
+    const publishEventResult = await eventsClient.publishEvent(cloudEvent)
+    logger.debug(`Publish event result: ${publishEventResult}`)
+    if (publishEventResult !== PUBLISH_EVENT_SUCCESS) {
+      logger.error(`Unable to publish event ${params.data.event}: Unknown event type`)
+      return errorResponse(HTTP_BAD_REQUEST, `Unable to publish event ${params.data.event}: Unknown event type`)
+    }
 
     logger.info(`Successful request: ${HTTP_OK}`)
 
-    return successResponse(event.type, {
+    return successResponse(params.data.event, {
```

Comment:
params.data.event is accessed a lot, maybe makes sense to assign it to a variable to make it more readable

Reply:
Sure!